website/source/api/kv.html.md

```diff
@@ -172,7 +173,7 @@ The table below shows this endpoint's support for
 
 ### Parameters
 
-- `key` `(string: "")` - Specifies the path of the key to read.
+- `key` `(string: "")` - Specifies the path of the key to create.
```

Comment:
For clarify, you could either rename this to say "create or update," or just simply say "Specifies the path of the key."

Reply:
Indeed, done